### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by anchormansam**
@EllerMel

### CODE REVIEW

File: src/Components/Accordion.js [hunk 1]
```diff
@@ -5,43 +5,39 @@ function Accordion() {
     const FoodDescription = [<Carousel />]
```
Comment (EllerMel) [inline src/Components/Accordion.js]: This doesn't appear to be used anywhere. Is it necessary?

File: src/Components/ApiCall.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        </div>
+                    </React.Fragment>
+                )
+        })
+        );
+    }
+}
+    
+    
+    export default ApiCall;
```
Comment (EllerMel) [inline src/Components/ApiCall.js]: Have a new line at the end of files. Not sure if it is an APAX thing, or an actual thing, but GitHub flagged it as well...

File: src/App.css [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 .jumbotron {
   background-image: url(/background.jpg);
-  margin-bottom: 0px;
+  margin-bottom: 0px !important;
   background-size: 100% 300%;
   background-repeat: no-repeat;
   background-position: 0px -158px;
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/App.css]: Good Catch! # has been removed

File: src/Components/Accordion.js [hunk 4]
```diff
@@ -5,43 +5,39 @@ function Accordion() {
...
             </div>
+        )
+    });
+
+
+    return (
+        <div id="accordion">
+            {accordionCards}
         </div>
     )
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/Accordion.js]: Sounds good. I'll look through all of the files and check for that. I have removed the one on this file specifically.

File: src/App.js [hunk 5]
```diff
@@ -7,6 +7,7 @@ import 'bootstrap/dist/js/bootstrap.bundle.min';
 import Jumbotron from './Components/Jumbotron';
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/App.js]: Lines have been removed!

File: src/Components/ApiCall.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    };
+
+    
+    componentDidMount() {
+        
+        Axios.get('https://entree-f18.herokuapp.com/v1/menu/12')
+        .then((Response) => {
+            let menu_items = Response.data.menu_items;
+            
+            console.log(Response.data.menu_items);
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/ApiCall.js]: Removed Console.

File: src/Components/ApiCall.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Axios from 'axios';
+import { privateEncrypt } from 'crypto';
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/ApiCall.js]: Line was commented out and found not to be needed (so far). I went ahead and deleted.

File: src/Components/ApiCall.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+                        
+                            <div className="carousel-caption d-md-block text-primary">
+                                <h2>{item.price}</h2>
+                                <h5>{item.title}</h5>
+                                <p>{item.description}</p>
+                            </div>
+                        </div>
+                    </React.Fragment>
+                )
+        })
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/ApiCall.js]: Fixed.

File: src/Components/Carousel.js [hunk 9]
```diff
@@ -1,46 +1,37 @@
 import React from 'react';
+import ApiCall from './ApiCall';
 
 
 
 const newImages = ['1.jpg', '2.jpg', '3.jpg',];
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/Carousel.js]: the newImages array was moved to another page to be used. I have updated.

File: src/Components/Carousel.js [hunk 10]
```diff
@@ -1,46 +1,37 @@
...
     <li data-target="#carouselExampleIndicators" data-slide-to="2"></li> */}
-    </ol>
-    <div className="carousel-inner">
-      <div className="carousel-item active">
-        <img className="d-block w-100" src="/images/sam.jpg" alt="First slide" />
-    </div>
-        {mappedImages}
+      </ol>
+      <div className="carousel-inner">
+        {/* <div className="carousel-item active">
```
[the remark this answers is not in the document]
Comment (anchormansam) [reply src/Components/Carousel.js]: Completed this as well as fixed the spacing.

File: src/Components/Accordion.js (hunk 1 above)
Comment (anchormansam) [reply src/Components/Accordion.js]: I have removed. Was being used but changed how the data was being called. Good Catch!

**Pull request closed without merging**